contributing/ci-ome-files.txt

```diff
@@ -0,0 +1,170 @@
+
+                #. |buildFilesSB|
+
+                See :jenkinsjob:`the build graph <OME-FILES-CPP-DEV-latest-superbuild/lastSuccessfulBuild/BuildGraph>`
+
+        :jenkinsjob:`OME-FILES-CPP-DEV-latest-win-superbuild`
+
+                This job builds the develop branches of OME Files components (Windows)
+
+                #. |buildFilesSB|
+
+                See :jenkinsjob:`the build graph <OME-FILES-CPP-DEV-latest-win-superbuild/lastSuccessfulBuild/BuildGraph>`
+
+        :jenkinsjob:`OME-FILES-CPP-DEV-latest-docs`
+
+                This job builds the documentation for develop branches of OME Files components
+
+                #. |buildFilesSB|
+
+                See :jenkinsjob:`the build graph <OME-FILES-CPP-DEV-latest-docs/lastSuccessfulBuild/BuildGraph>`
+
+
+        :jenkinsjob:`OME-FILES-CPP-DEV-merge-trigger`
+
+                This job runs the daily OME Files jobs used for reviewing the PRs
+                opened against the develop branches of OME Files.
+
+                #. Triggers :term:`BIOFORMATS-DEV-merge-push`
+                #. Triggers :term:`OME-FILES-CPP-DEV-merge-push-superbuild`
+                #. Triggers downstream merge projects
+
+                See :jenkinsjob:`the build graph <OME-FILES-CPP-DEV-merge-trigger/lastSuccessfulBuild/BuildGraph>`
+
+        :jenkinsjob:`OME-FILES-CPP-DEV-merge-push-superbuild`
+
+                This job merges all the PRs opened against develop for each OME Files component
+
+                #. |merge|
+                #. Pushes the branch to :bf_scc_branch:`develop/merge/files-cpp`
+
+        :jenkinsjob:`OME-FILES-CPP-DEV-merge`
+
+                This job builds the merge branches of OME Files components (Unix)
+
+                #. Checks out :bf_scc_branch:`develop/merge/files-cpp`
+                #. |buildFiles|
+
+        :jenkinsjob:`OME-FILES-CPP-DEV-merge-sourcebuild`
+
+                This job builds the merge branch of
+                ``ome-cmake-superbuild`` using the latest source
+                release of all OME Files components (Unix)
+
+                #. Checks out :bf_scc_branch:`develop/merge/files-cpp`
+                #. |buildFilesSB|
+
+        :jenkinsjob:`OME-FILES-CPP-DEV-merge-superbuild`
+
+                This job builds the merge branches of OME Files components (Unix)
+
+                #. Checks out :bf_scc_branch:`develop/merge/files-cpp`
+                #. |buildFilesSB|
+
+        :jenkinsjob:`OME-FILES-CPP-DEV-merge-win-superbuild`
+
+                This job builds the merge branches of OME Files components (Windows)
+
+                #. Checks out :bf_scc_branch:`develop/merge/files-cpp`
+                #. |buildFilesSB|
+
+        :jenkinsjob:`OME-FILES-CPP-DEV-merge-docs`
+
+                This job builds the documentation for merge branches of OME Files components
+
+                #. |buildFilesSB|
+
+            See :jenkinsjob:`the build graph <OME-FILES-CPP-DEV-merge-docs/lastSuccessfulBuild/BuildGraph>`
```

Comment:
Sorry @rleigh-codelibre one more you need to indent further here. Once that's fixed I'll merge, thanks